pigeon/ui/areas/controlsarea.py

```diff
@@ -53,6 +54,9 @@ def __init__(self, *args, **kwargs):
         self.last_message_received_time = None
         self.uav_pictures_taken = ""
         self.uav_pictures_transmitted = ""
+        self.reconnectBtn = QtWidgets.QPushButton("Reonnect")
```

Comment:
Just a small typo. I think your meant "Reconnect".

Reply:
> Just a small typo. I think your meant "Reconnect".

Changed button name to "Reconnect".